plugins/linguist-backend/src/api/LinguistBackendApi.ts

```diff
@@ -98,7 +106,8 @@ export class LinguistBackendApi {
       fields: ['kind', 'metadata'],
     };
 
-    const response = await this.catalogClient.getEntities(request);
+    const { token } = await this.tokenManager.getToken();
+    const response = await this.catalogClient.getEntities(request, { token });
```

Comment:
if you do make the token manager an optional thing to supply (which actually can be helpful in tests too i guess), you could make it like

```ts
const tr = await this.tokenManager?.getToken();
const response = await this.catalogClient.getEntities(request, { token: tr?.token });
```